index.html

```diff
@@ -66,6 +66,157 @@ <h1>
             <img src="./assest/Disabled.svg" alt="disabled button">
         </button>
     </section>
+    <section class="works">
+        <h1>My Recent Works</h1>
+        <article class="card-work">
+            <div class="card-image">
+                <img src="https://source.unsplash.com/327x474/?cat,library" alt="image work">
+            </div>
+            <h2 class="card-title">
+                Multi-Post Stories Gain+Glory
+            </h2>
+            <div class="card-tags">
+                <p>Rubi on rails</p>
+                <p>css</p>
+                <p>JavaScript</p>
+                <p>html</p>
+            </div>
+            <button type="button">
+                See Project
+            </button>
+        </article>
+        <article class="card-work">
+            <div class="card-image">
+                <img src="https://source.unsplash.com/327x474/?cat,library" alt="image work">
+            </div>
+            <h2 class="card-title">
+                Multi-Post Stories Gain+Glory
+            </h2>
+            <div class="card-tags">
+                <p>Rubi on rails</p>
+                <p>css</p>
+                <p>JavaScript</p>
+                <p>html</p>
+            </div>
```

Comment:
- [ ] Consider using a `ul` tag to wrap the project technologies and each of the technology is a `li` tag.
This is because the project technologies is a list of unordered items and it would be semantically correct to wrap them in a `ul` tag. read more about ul tags [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul)

```suggestion
            <ul class="card-tags">
                <li>Rubi on rails</li>
                <li>css</li>
                <li>JavaScript</li>
                <li>html</li>
            </ul>
```